ember/app/router.js

```diff
@@ -50,6 +50,7 @@ Router.map(function() {
   this.route('tracking', function() {
     this.route('info');
     this.route('details', { path: '/:user_ids' });
+    this.route('redirect', { path: '/:user_ids/map' });
```

Comment:
I'd prefer it if we would name the route e.g. `map-redirect` to make it clear which URL this route handles